Makefile

```
@@ -27,7 +27,7 @@ SILENT ?= @
 UNIT_TEST_IGNORE := "stackrox/rox/sensor/tests|stackrox/rox/operator/tests|stackrox/rox/central/reports/config/store/postgres|stackrox/rox/central/complianceoperator/v2/scanconfigurations/store/postgres|stackrox/rox/central/auth/store/postgres|stackrox/rox/scanner/e2etests"
 
 ifeq ($(TAG),)
-TAG=$(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*')
+TAG=$(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*' | xargs $(BASE_DIR)/tools/tag.py)
```

Just patching `.x` -> `.0` can be done similarly to the [operator](https://github.com/stackrox/stackrox/blob/980daf6fdcc9ac9ef52895be414518eb1c895bd4/operator/Makefile#L8), with `sed`:
```suggestion
TAG=$(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*' | sed -E 's@^(([[:digit:]]+\.)+)x(-)?@\10\3@')
```